lib/elixir/lib/version.ex

```diff
@@ -111,11 +111,27 @@ defmodule Version do
   end
 
   defmodule InvalidRequirementError do
-    defexception [:message]
+    defexception [:requirement]
+
+    def exception(requirement) do
```

Comment:
I don't think this clause is required at all then. Is it? Same to the one below.